prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -298,10 +311,10 @@ class Simulator : CoolTest() {
             true
         )
         val simpleReport = Simulation( // Simulate a single large SimpleReport Send
-            "simpleReport: One submission SimpleReport data, 1000 Items",
+            "simpleReport: One submission SimpleReport data, 3000 Items",
             1,
```

Comment:
Current submissions from simple_report max out around 1000 rows.   So I amped this up to 3000 rows, to stay ahead.